src/org/isf/lab/gui/LabEditExtended.java

```diff
@@ -151,12 +155,13 @@ private void fireLabUpdated() {
 	
 	//private LabManager labManager = new LabManager(Context.getApplicationContext().getBean(LabIoOperations.class));
 	private LabManager labManager = Context.getApplicationContext().getBean(LabManager.class);
+	private PrintManager printManager = Context.getApplicationContext().getBean(PrintManager.class);
 	private LabRowManager lRowManager = Context.getApplicationContext().getBean(LabRowManager.class);
 	private AdmissionBrowserManager admMan = Context.getApplicationContext().getBean(AdmissionBrowserManager.class);
 	private ExamRowBrowsingManager rowManager = Context.getApplicationContext().getBean(ExamRowBrowsingManager.class);
 	
 	public LabEditExtended(JFrame owner, Laboratory laboratory, boolean inserting) {
-		super(owner, true);
+		//super(owner, true);
```

Comment:
This change removes the modality, so the calling window can go over the called one, but it is needed becuase ModalJFrame is a JFrame, so we miss the possibility to specify an owner

But the ModalJFrame class has a method for that, that is showAsModal() that should be called instead of setVisible(true) in the calling (LabBrowser) class that I cannot see modified here.

Reply:
Fixed with the push c9b6c019656dba1594baf288c36cf9d381f4130d